torch/nn/modules/activation.py

```diff
@@ -763,10 +763,8 @@ def extra_repr(self) -> str:
 
 
 class LogSigmoid(Module):
-    r"""Applies the element-wise function:
-
-    .. math::
-        \text{LogSigmoid}(x) = \log\left(\frac{ 1 }{ 1 + \exp(-x)}\right)
+    r"""Applies the LogSigmoid function :math:`\text{LogSigmoid}(x) =
+    \log\left(\frac{ 1 }{ 1 + \exp(-x)}\right)` element-wise:
```

Comment:
I think the formula with `.. math::` is better as it is easier to read than the inline version.
This PR: https://docs-preview.pytorch.org/71099/generated/torch.nn.LogSigmoid.html?highlight=logsigmoid#torch.nn.LogSigmoid
Current: https://pytorch.org/docs/stable/generated/torch.nn.LogSigmoid.html?highlight=logsigmoid#torch.nn.LogSigmoid

Reply:
yeah, I have doubts about this one myself, will fix it back